snap/local/runtime-helpers/bin/start-consul.sh

```diff
@@ -15,16 +15,21 @@ cat > "$SNAP_DATA/consul/config/consul_default.json" <<EOF
 }
 EOF
 
-echo "$(date) deploying additional ACL configuration for Consul"
-cat > "$SNAP_DATA/consul/config/consul_acl.json" <<EOF
+acls=${EDGEX_SECURITY_SECRET_STORE:-true}
+logger "start-consul.sh: acls=$acls"
+
+if [ "$acls" == "true" ]; then
+    echo "$(date) deploying additional ACL configuration for Consul"
+    cat > "$SNAP_DATA/consul/config/consul_acl.json" <<EOF
 {
-    "acl": {
+      "acl": {
       "enabled": true,
       "default_policy": "deny",
       "enable_token_persistence": true
     }
 }
 EOF
+fi
```

Comment:
Can be done in another PR: Since this PR makes the services start after configuration, it makes sense to move similar configuration logic to the configure hook. Removing of the acl file already does happen in the configure hook: https://github.com/edgexfoundry/edgex-go/blob/1839698a2dde2b913a8fb2ca4dd0c55fce4f3cab/snap/local/hooks/cmd/configure/configure.go#L189-L195

Reply:
@farshidtz can you open an an issue on our internal board for this?